src/main/java/com/softserve/controller/GroupController.java

```diff
@@ -35,7 +35,9 @@ public GroupController(GroupService groupService, GroupMapper groupMapper) {
     @ApiOperation(value = "Get the list of all groups")
     public ResponseEntity<List<GroupDTO>> getAll() {
         log.info("In getAll ()");
-        return ResponseEntity.status(HttpStatus.OK).body(groupMapper.groupsToGroupDTOs(groupService.getAll()));
+        return ResponseEntity
+                .status(HttpStatus.OK)
+                .body(groupMapper.groupsToGroupDTOs(groupService.getAllBySortingOrder()));
```

Comment:
I don't think it's a good idea to force sorting by order in any case, it should rather be a default sort if none specified